ha-example/README.md

```diff
@@ -0,0 +1,198 @@
+# Flink cluster High Availability configuration
+
+A Flink cluster deployment consists of operators, job managers, and task managers.
+However, the deployment is not configured to ensure high availability (HA) by default, additional configuration is needed to achieve it.
+
+## Flink Kubernetes Operator
+
+The Flink Kubernetes operator manages the Flink cluster Deployments.
+It supports high availability through adding a standby operator instance and using leader election functionality to ensure that only one instance is _in charge_ of the Flink cluster Deployments at any one time.
+To enable leader election, we need to add the following two mandatory parameters to the Kubernetes operator's configuration.
+
+```
+kubernetes.operator.leader-election.enabled: true
+kubernetes.operator.leader-election.lease-name: flink-operator-lease
+```
+The lease name must be unique in the deployed namespace.
+
+When installing the Flink Kubernetes operator using `helm`, we need to add these 2 parameters into the existing default configuration string.
+This can be done via the command line using the `--set` flag:
+```
+helm install flink-kubernetes-operator flink-operator-repo/flink-kubernetes-operator \
+--set podSecurityContext=null \
+--set defaultConfiguration."log4j-operator\.properties"=monitorInterval\=30 \
+--set defaultConfiguration."log4j-console\.properties"=monitorInterval\=30 \
+--set replicas=2 \
+--set defaultConfiguration."flink-conf\.yaml"="kubernetes.operator.metrics.reporter.slf4j.factory.class\:\ org.apache.flink.metrics.slf4j.Slf4jReporterFactory
+kubernetes.operator.metrics.reporter.slf4j.interval\:\ 5\ MINUTE
+kubernetes.operator.reconcile.interval:\ 15\ s
+kubernetes.operator.observer.progress-check.interval:\ 5\ s
```

Comment:
Are these configurations somehow related to HA configuration (except replicas)? It doesn't look like yes to me and it might be confusing for anyone.